### PR TITLE
chore: update links to pt-br learning area to learning-area

### PR DESCRIPTION
**Pull request by schalkneethling**
The pt-br version of the learning area repository has been archived.
https://github.com/orgs/mdn/teams/yari-content-pt-br/discussions/2

This updates links to point to the same pages on mdn/learning-area

### CODE REVIEW

File: files/pt-br/learn/javascript/first_steps/a_first_splash/index.html [hunk 1]
```diff
@@ -674,6 +674,6 @@ <h3 id="Brincando_com_objetos_do_navegador">Brincando com objetos do navegador</
 
 <h2 id="Finalizado_por_enquanto...">Finalizado por enquanto...</h2>
 
-<p>Então é isso, para construir o exemplo — você chegou ao final, muito bem! Teste o resultado do seu código, ou <a href="https://mdn.github.io/learning-area-pt-br/javascript/introduction-to-js-1/first-splash/jogo-advinhe-o-numero.html">jogue com nossa versão finalizada aqui</a>. Se você não conseguir fazer o exemplo funcionar, compare com o <a href="https://github.com/mdn/learning-area-pt-br/blob/master/javascript/introduction-to-js-1/first-splash/jogo-advinhe-o-numero.html">código fonte</a>.</p>
+<p>Então é isso, para construir o exemplo — você chegou ao final, muito bem! Teste o resultado do seu código, ou <a href="https://github.com/mdn/learning-area/blob/main/javascript/introduction-to-js-1/first-splash/number-guessing-game.html">jogue com nossa versão finalizada aqui</a>. Se você não conseguir fazer o exemplo funcionar, compare com o <a href="https://github.com/mdn/learning-area/blob/master/javascript/introduction-to-js-1/first-splash/number-guessing-game.html">código fonte</a>.</p>
```
Comment (schalkneethling) [inline files/pt-br/learn/javascript/first_steps/a_first_splash/index.html:677]:
```suggestion
<p>Então é isso, para construir o exemplo — você chegou ao final, muito bem! Teste o resultado do seu código, ou <a href="https://mdn.github.io/learning-area/javascript/introduction-to-js-1/first-splash/number-guessing-game.html">jogue com nossa versão finalizada aqui</a>. Se você não conseguir fazer o exemplo funcionar, compare com o <a href="https://mdn.github.io/learning-area/javascript/introduction-to-js-1/first-splash/number-guessing-game.html">código fonte</a>.</p>
```

File: files/pt-br/learn/javascript/first_steps/a_first_splash/index.html [hunk 2]
```diff
@@ -674,6 +674,6 @@ <h3 id="Brincando_com_objetos_do_navegador">Brincando com objetos do navegador</
 
 <h2 id="Finalizado_por_enquanto...">Finalizado por enquanto...</h2>
 
-<p>Então é isso, para construir o exemplo — você chegou ao final, muito bem! Teste o resultado do seu código, ou <a href="https://mdn.github.io/learning-area-pt-br/javascript/introduction-to-js-1/first-splash/jogo-advinhe-o-numero.html">jogue com nossa versão finalizada aqui</a>. Se você não conseguir fazer o exemplo funcionar, compare com o <a href="https://github.com/mdn/learning-area-pt-br/blob/master/javascript/introduction-to-js-1/first-splash/jogo-advinhe-o-numero.html">código fonte</a>.</p>
+<p>Então é isso, para construir o exemplo — você chegou ao final, muito bem! Teste o resultado do seu código, ou <a href="https://mdn.github.io/learning-area/javascript/introduction-to-js-1/first-splash/number-guessing-game.html">jogue com nossa versão finalizada aqui</a>. Se você não conseguir fazer o exemplo funcionar, compare com o <a href="https://mdn.github.io/learning-area/javascript/introduction-to-js-1/first-splash/number-guessing-game.html">código fonte</a>.</p>
```
Comment (schalkneethling) [inline files/pt-br/learn/javascript/first_steps/a_first_splash/index.html:677]:
```suggestion
<p>Então é isso, para construir o exemplo — você chegou ao final, muito bem! Teste o resultado do seu código, ou <a href="https://mdn.github.io/learning-area/javascript/introduction-to-js-1/first-splash/number-guessing-game.html">jogue com nossa versão finalizada aqui</a>. Se você não conseguir fazer o exemplo funcionar, compare com o <a href="https://github.io/mdn/learning-area/blob/main/javascript/introduction-to-js-1/first-splash/number-guessing-game.html">código fonte</a>.</p>
```

File: files/pt-br/learn/javascript/first_steps/a_first_splash/index.html [hunk 3]
```diff
@@ -674,6 +674,6 @@ <h3 id="Brincando_com_objetos_do_navegador">Brincando com objetos do navegador</
 
 <h2 id="Finalizado_por_enquanto...">Finalizado por enquanto...</h2>
 
-<p>Então é isso, para construir o exemplo — você chegou ao final, muito bem! Teste o resultado do seu código, ou <a href="https://mdn.github.io/learning-area-pt-br/javascript/introduction-to-js-1/first-splash/jogo-advinhe-o-numero.html">jogue com nossa versão finalizada aqui</a>. Se você não conseguir fazer o exemplo funcionar, compare com o <a href="https://github.com/mdn/learning-area-pt-br/blob/master/javascript/introduction-to-js-1/first-splash/jogo-advinhe-o-numero.html">código fonte</a>.</p>
+<p>Então é isso, para construir o exemplo — você chegou ao final, muito bem! Teste o resultado do seu código, ou <a href="https://mdn.github.io/learning-area/javascript/introduction-to-js-1/first-splash/number-guessing-game.html">jogue com nossa versão finalizada aqui</a>. Se você não conseguir fazer o exemplo funcionar, compare com o <a href="https://github.io/mdn/learning-area/blob/main/javascript/introduction-to-js-1/first-splash/number-guessing-game.html">código fonte</a>.</p>
```
Comment (schalkneethling) [inline files/pt-br/learn/javascript/first_steps/a_first_splash/index.html:677]:
```suggestion
<p>Então é isso, para construir o exemplo — você chegou ao final, muito bem! Teste o resultado do seu código, ou <a href="https://mdn.github.io/learning-area/javascript/introduction-to-js-1/first-splash/number-guessing-game.html">jogue com nossa versão finalizada aqui</a>. Se você não conseguir fazer o exemplo funcionar, compare com o <a href="https://github.com/mdn/learning-area/blob/main/javascript/introduction-to-js-1/first-splash/number-guessing-game.html">código fonte</a>.</p>
```

**Review by josielrocha — APPROVED** [review]
LGTM ;)

**Pull request merged**